Dockerfile

```diff
@@ -13,7 +13,13 @@ COPY config/settings-reducer.yaml.template /app/config/settings-reducer.yaml
 COPY $REQUIREMENTS /app/config/requirements.txt
 
 # Install developer tools (needed for psutil)
-RUN apt-get update && apt-get install -y python3-dev gcc
+RUN apt-get update && apt-get install -y python3-dev gcc wget
+
+# Install grpc health probe checker
+RUN GRPC_HEALTH_PROBE_VERSION=v0.4.24 && \
```

Comment:
Ok, but we won't need this binary for k8s. It's good for compose but other than that it just add more maintenance. For example, this version maybe need to be in sync with grpcio version.